scripts/incentive-spreadsheet-to-json.ts

```diff
@@ -19,6 +20,11 @@ async function convertToJson(
   file: IncentiveFile,
   strict: boolean,
 ) {
+  if (strict && !(state in LOW_INCOME_THRESHOLDS_BY_AUTHORITY)) {
```

Comment:
so, for my understanding, in order to use strict mode, low income thresholds must be defined? What about for states that have no low income incentives? (I'm not sure if this is a possible scenario but dont think there's anything state-wide that makes it a requirement)

Reply:
I wanted to gate this behavior and thought the overlap with using strict in terms of use cases was sufficient to just lump them together. I think most people who want to use one will use the other. But there's no reason they can't be decoupled.

I'll add a separate arg to turn the low-income functionality on/off.